README.md

```diff
@@ -47,6 +47,13 @@ On top of that, every commit is tested using an address sanitizer in CI to catch
 
 Even though neither fuzzing not sanitization has revealed any safety bugs yet, please don't hesitate to file an issue if you run into any crashes or other unexpected behaviour.
 
+## `no_std`
```

Comment:
It looks like this paragraph needs some changes to reflect the new feature flag approach.